src/ocean/io/compress/Lzo.d

```diff
@@ -243,6 +243,23 @@ class Lzo
     }
 }
 
+/// Simple compress / decompress example.
+unittest
+{
+    auto lzo = new Lzo;
+    auto original = "compress this!".dup;
+
+    // Compress.
+    auto compressed = new void[lzo.maxCompressedLength(original.length)];
```

Comment:
Not very important, but without the `.dup` above `compressed` and `uncompressed` can be static `ubyte[n]` arrays.

Reply:
I think that's less intuitive as a usage example.